spyderlib/plugins/editor.py

```diff
@@ -944,7 +952,7 @@ def get_plugin_actions(self):
         self.recent_file_menu = QMenu(_("Open &recent"), self)
         self.recent_file_menu.aboutToShow.connect(self.update_recent_file_menu)
 

```

Comment:
Yeah, that makes more sense to me (the context menu of the editor), what does Sublime do haha @blink1073?